src/settings/JobProfiles/JobProfileDetailsRoute/JobProfileDetailsRoute.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  get,
+  find,
+} from 'lodash';
+
+import { JobProfileDetails } from '../JobProfileDetails';
+
+const JobProfileDetailsRoute = ({
+  resources: {
+    jobProfile,
+    mappingProfile,
+  },
+  history,
+  location,
+  match,
+}) => {
+  // `find` is used to make sure that the job profile that matches the id in url is displayed
+  const jobProfileRecord = find([get(jobProfile, 'records.0', {})], { id: match.params.id });
+  const mappingProfileRecord = find([get(mappingProfile, 'records.0', {})], { id: jobProfileRecord?.mappingProfileId });
+
+  return (
+    <JobProfileDetails
+      jobProfile={jobProfileRecord}
+      mappingProfile={mappingProfileRecord}
+      loading={!jobProfileRecord || !mappingProfileRecord}
+      onCancel={() => history.push(`/settings/data-export/job-profiles${location.search}`)}
+    />
+  );
+};
+
+JobProfileDetailsRoute.propTypes = {
+  history: PropTypes.shape({ push: PropTypes.func.isRequired }).isRequired,
+  location: PropTypes.shape({ search: PropTypes.string }).isRequired,
+  match: PropTypes.shape({ params: PropTypes.shape({ id: PropTypes.string }) }).isRequired,
+  resources: PropTypes.shape({
+    jobProfile: PropTypes.shape({}),
+    mappingProfile: PropTypes.shape({}),
+  }).isRequired,
+};
+
+JobProfileDetailsRoute.manifest = Object.freeze({
+  jobProfile: {
+    type: 'okapi',
+    path: 'data-export/jobProfiles/:{id}',
+    throwErrors: false,
+  },
+  mappingProfile: {
+    type: 'okapi',
+    throwErrors: false,
```

Comment:
if the error throw is disabled. Are there any ways to notify users about the loading problems?

Reply:
I think it will be part of the future story to add error callouts.

Reply:
It sounds good, but I think we should throw error for now if there are not callouts. As far as I remember, this approach in used in a lot of places. Or I misunderstand the idea?

Reply:
That is right. No objections.